tasks/configure_namespace.yml

```diff
@@ -114,8 +114,8 @@
       metadata:
         name: pipelinerun-cleaner
       spec:
-        # every 5 min
-        schedule: "*/5 * * * *"
+        # Every day at midnight
+        schedule: "{{ pipeline_run_pruning_schedule }}"
```

Comment:
Should role define default?
Should it be documented?

Reply:
We don't use default to fail horribly when groupvars are not provided.
Yes nice catch, I documented it in README.